utils/build.config

```diff
@@ -6,7 +6,7 @@ ARGOBOTS = 9d48af08403bac649598942b5ee848c14600cd8a
 CART = 7bde2eaec684faa02372caca464b96136348aad4
 PMDK = 1.5.1
 ISAL = v2.26.0
-SPDK = 051297114cb393d3eb1169520d474e81b4215bf0
+SPDK = v19.04.1
```

Comment:
Nice that we can use an upstream release here.

But landing this will need to coordinate with an spdk RPM update so that the RPM build of daos doesn't break due to building with the older spdk RPM.

Reply:
Yes, could you help me to handle the RPM update and coordinate the landings? Thanks.

Reply:
I have a [PR in progress](https://github.com/daos-stack/spdk/pull/9) but it's competing with a number of other tasks that I have on my plate so is getting time when I am blocked on all of my other tasks.

Reply:
Brian, is the collaborative landings still in progress? If you need any help from me (like refreshing this patch), please just let me know. Thanks.